lib/models/live_stream_model.dart

```diff
@@ -0,0 +1,41 @@
+class LiveStreamModel {
+  LiveStreamModel({
+    required this.liveStreamId,
+    required this.title,
+    required this.startTime,
+    required this.viewers,
+    required this.channelId,
+    required this.directorId,
+    required this.directorName,
+  });
```

Comment:
```dart
// ignore_for_file: sort_constructors_first
```
Instead of having that ignore, we can move `LiveStreamModel()` and `LiveStreamModel.fromSnapshot()` to the top of the class.